### PR TITLE
Fix failing EasyStandard build

### PR DESCRIPTION
**Pull request by TomasVotruba**
Fix for https://github.com/eonx-com/easy-monorepo/runs/1212352031

image

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Tip for colros in CI :) 
https://tomasvotruba.com/blog/2020/09/28/how-to-add-colors-to-continuious-integration-output/

File: .github/workflows/split_tests.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ jobs:
                     restore-keys: ${{ matrix.php }}-comp-
 
             -   run: composer global require hirak/prestissimo
-            -   run: composer install --prefer-dist --no-progress
-            -   run: vendor/bin/monorepo-builder localize-composer-paths
-            -   run: cd packages/${{ matrix.package }}; composer update --prefer-dist --no-progress
+            -   run: composer install --prefer-dist --no-progress --ansi
+            -   run: vendor/bin/monorepo-builder localize-composer-paths --ansi
+            -   run: cd packages/${{ matrix.package }}; composer update --prefer-dist --no-progress --ansi
```
Comment (TomasVotruba) [inline .github/workflows/split_tests.yml:50]: Tip for colors in CI :) 
https://tomasvotruba.com/blog/2020/09/28/how-to-add-colors-to-continuious-integration-output/

**Comment by TomasVotruba** [comment]
CI now passes :partying_face: 

Ready to merge :+1:

**Comment by natepage** [comment]
Thanks @TomasVotruba

**Pull request merged**